data/influxdb_urls.yml

```diff
@@ -23,6 +23,9 @@ cloud:
         - name: EU Frankfurt
           location: Frankfurt, Germany
           url: https://eu-central-1-1.aws.cloud2.influxdata.com
+        - name: Asia Pacific (Australia)
+          location: Sydney, Australia
+          url: https://ap-southeast-2-1.aws.cloud2.influxdata.com
```

Comment:
Is this region currently in beta? If so, add:

```yaml
status: beta
```

Reply:
Thank you @sanderson, region is officially GA